### DISCUSSION TITLE
How do I set vim.opt.colorcolumn to only appear in certain file types?

### DISCUSSION THREAD

**Discussion by freightheave** [neovim/neovim#29664] [Q&A] [answered]
So I use `startup.nvim` as a splash screen when I open neovim, but `vim.opt.colorcolumn` is my visual guide I have set at col = 80, how do I make it so that it doesn't appear in the splash screen like so?

![Screenshot_20240712_002206](https://github.com/neovim/neovim/assets/17759116/70d07d22-dc1a-476d-a28f-783f9fe8aee2)

**Comment by zeertzjq** [comment, marked as answer]
See `:h ftplugin-overrule`

**Reply by freightheave** [reply]
Thank you for the help, I didn't completely understand the section, but I will play around with the `overrule settings after loading the global plugin` section and find out.
